recipe/meta.yaml

```
@@ -1,8 +1,9 @@
 {% set version = "6.0.2" %}
 {% set python_min = "3.8" %}
+{% set build_number = 4 %}
```

```suggestion
{% set build = 4 %}
```

otherwise the bot has a hard time.